app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError,\
+    NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` should include a descriptive message, such as 'Visitor is not vaccinated.'